src/main/java/Task.java

```diff
@@ -0,0 +1,44 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+
+    }
+
+    public Task(String description, boolean done) {
+        this.description = description;
+        this.isDone = done; 
+    }
+
+    public String getStatusIcon() {
+        return "[" + (isDone ? "X" : " ") + "]"; // mark done task with X
+    }
+
+    public String getName() {
+        return this.description;
+    }
+    
+    public void markAsDone() {
+        this.isDone = true;
+        
+        
```

Comment:
Maybe can consider removing unused line spacing...same goes for some other places